### PR TITLE
Various UX improvements

### PR DESCRIPTION
**Pull request by t2d**
Give early feedback to prevent me from running in the wrong direction :)

### CODE REVIEW

File: default_translations/de/messages.de.yml [hunk 1]
```diff
@@ -5,16 +5,24 @@ start:
...
 index:
   title: Verwalte deinen E-Mail Account
   voucher-headline: Deine Einladungscodes
   voucher-limit: Du hast keine Einladungscodes mehr verfügbar.
   voucher-disable: Du bekommst am %date% Uhr drei Einladungscodes gutgeschrieben. Dies dient der Vermeidung von Massenregistrierungen und Spam.
+  voucher-explanation: >
+    Nutze deine Einladungscodes um Freundinnen und Freunde zu diesem Dienst einzuladen.
+    Standardmäßig kannst du bis zu drei Freundinnen einladen.
+    Wenn du mehr einladen möchtest, wende dich an die Admins.
+    Wir speichern Einladungsdaten für einen begrenzten Zeitraum um Ausbeutung unserer Dienste zu verhindern.
```
Comment (0x46616c6b) [inline default_translations/de/messages.de.yml]:
```suggestion
    Wir speichern Einladungsdaten für einen begrenzten Zeitraum um Missbrauch unserer Dienste zu verhindern.
```

File: default_translations/en/messages.en.yml [hunk 2]
```diff
@@ -5,17 +5,24 @@ start:
...
 index:
   title: Manage your email-account
   voucher-headline: Your invite codes
   voucher-limit: You don't have any invite codes left.
   voucher-disable: At %date% you will receive three invite codes. This is to avoid mass registrations and spam.
+  voucher-explanation: >
+    Use your invite codes to invite friends to this service.
+    By default, you can invite a maximum of three friends.
+    If you want to invite more, please contact the administrators.
+    We store invite data for a limited amount of time to protect against exploitation of our resources.
```
Comment (0x46616c6b) [inline default_translations/en/messages.en.yml]:
```suggestion
    We store invite data for a limited amount of time to protect against abuse of our services.
```

File: templates/Start/vouchers.html.twig [hunk 3]
```diff
@@ -42,5 +42,10 @@
                 {% endif %}
             {%  endif %}
         </div>
+        <div class="col-xs-12 col-sm-6 col-md-6">
```
Comment (0x46616c6b) [inline templates/Start/vouchers.html.twig]: That should be under the headline, so you should use a full width row for headline and a new row with two columns for the content and the description.

File: templates/Start/aliases.html.twig [hunk 4]
```diff
@@ -11,61 +11,45 @@
...
 
             {%  if alias_creation_custom %}
                 {{ form_start(custom_alias_form) }}
                 {{ form_errors(custom_alias_form) }}
                 <div class="form-group">
-                    {{ form_label(custom_alias_form.alias) }}
                     <div class="input-group">
                         {{ form_widget(custom_alias_form.alias, {'attr': {'class': 'form-control' }, 'value': ''}) }}
                         <span class="input-group-addon">@{{ user_domain }}</span>
+                        <div class="input-group-btn">
```
Comment (0x46616c6b) [inline templates/Start/aliases.html.twig]: Awesome! 👍

File: default_translations/de/messages.de.yml [hunk 5]
```diff
@@ -5,16 +5,24 @@ start:
   registration-text: Du hast noch keinen E-Mail Account bei %domain%? Kein Problem! Wenn eine Freundin oder ein Freund einen Account besitzt, kann diese dir einen Einladungscode zukommen lassen. Mit diesem kannst du dir sofort einen Account registrieren.
   registration-button: Account anlegen
   account-settings: Account verwalten
+  account-settings-desc: Ändere Passwörter und mehr
   account-delete: Account löschen
   aliases: Alias-Adressen
   aliases-delete: Alias-Adresse löschen
+  aliases-desc: Verschleiere deine Identität
   vouchers: Einladungscodes
+  vouchers-desc: Lade Freunde zu diesem Dienst ein
```
Comment (doobry-systemli) [inline default_translations/de/messages.de.yml]: I would shorten this: "Lade Freund*innen ein"

File: default_translations/de/messages.de.yml [hunk 6]
```diff
@@ -5,16 +5,24 @@ start:
   registration-text: Du hast noch keinen E-Mail Account bei %domain%? Kein Problem! Wenn eine Freundin oder ein Freund einen Account besitzt, kann diese dir einen Einladungscode zukommen lassen. Mit diesem kannst du dir sofort einen Account registrieren.
   registration-button: Account anlegen
   account-settings: Account verwalten
+  account-settings-desc: Ändere Passwörter und mehr
```
Comment (doobry-systemli) [inline default_translations/de/messages.de.yml]: Shortened: "Passwörter und mehr"

File: default_translations/en/messages.en.yml [hunk 7]
```diff
@@ -5,17 +5,24 @@ start:
   registration-text: You don't have an email-account with %domain% yet? No problem! If one of your friends already has an account, you can get an invite code from them and immediately set up your own account.
   registration-button: Create account
   account-settings: Account Settings
-  account-delete: Account löschen
+  account-settings-desc: Change passwords and more
```
Comment (doobry-systemli) [inline default_translations/en/messages.en.yml]: "Passwords and more"

File: default_translations/en/messages.en.yml [hunk 8]
```diff
@@ -5,17 +5,24 @@ start:
...
+  account-delete: Delete account
   aliases: Alias addresses
   aliases-delete: delete alias address
+  aliases-desc: Hide your identity to other parties
   vouchers: Invite codes
-
+  vouchers-desc: Invite friends to this service
```
Comment (doobry-systemli) [inline default_translations/en/messages.en.yml]: "Invite your friends"

File: default_translations/de/messages.de.yml [hunk 9]
```diff
@@ -5,16 +5,24 @@ start:
...
 index:
   title: Verwalte deinen E-Mail Account
   voucher-headline: Deine Einladungscodes
   voucher-limit: Du hast keine Einladungscodes mehr verfügbar.
   voucher-disable: Du bekommst am %date% Uhr drei Einladungscodes gutgeschrieben. Dies dient der Vermeidung von Massenregistrierungen und Spam.
+  voucher-explanation: >
+    Nutze deine Einladungscodes um Freundinnen und Freunde zu diesem Dienst einzuladen.
+    Standardmäßig kannst du bis zu drei Freundinnen einladen.
+    Wenn du mehr einladen möchtest, wende dich an die Admins.
+    Wir speichern Einladungsdaten für einen begrenzten Zeitraum um Missbrauch unserer Dienste zu verhindern.
```
Comment (doobry-systemli) [inline default_translations/de/messages.de.yml]: "Nutze deine Einladungscodes<KOMMA> um Freund*innen hierher einzuladen. Im Normalfall hast du drei Einladungen zur Verfügung. Wenn du mehr brauchst, melde dich bei den Admins. Wir speichern Einladungsdaten für einen begrenzten Zeitraum<KOMMA> um Missbrauch zu verhindern."

File: templates/Start/vouchers.html.twig [hunk 10]
```diff
@@ -10,9 +10,11 @@
 {% endblock %}
 {% block content %}
     <div class="row">
-        <div class="col-xs-12 col-sm-6 col-md-6">
+        <div class="col-xs-12 col-sm-12 col-md-12">
             <h3>{{ "index.voucher-headline"|trans }}</h3>
-            {% if domain == user_domain %}
+        </div>
+        <div class="col-xs-12 col-sm-6 col-md-6">
+        {% if domain == user_domain %}
```
Comment (doobry-systemli) [inline templates/Start/vouchers.html.twig]: Indention needs to be fixed.

File: templates/Start/index.html.twig [hunk 11]
```diff
@@ -24,6 +24,7 @@
                 <img src="{{  asset('build/images/diff-added.svg') }}" alt="Invite Codes">
                 <div class="caption">
                     <h3>{{ "start.vouchers"|trans }}</h3>
+                    <p>{{ "start.vouchers-desc"|trans }}</p>
```
Comment (doobry-systemli) [inline templates/Start/index.html.twig]: Should be `<p class="alert alert-info">`, no?

File: default_translations/de/messages.de.yml (hunk 5 above)
Comment (t2d) [reply default_translations/de/messages.de.yml]: > The invite code infobox looks ugly, they're missing the `class= "alert alert-info"`.

I don't agree. This isn't an alert. Maybe just add a border to this without coloring?

File: templates/Start/index.html.twig (hunk 11 above)
Comment (t2d) [reply templates/Start/index.html.twig]: No, this is the subtitle of the voucher feature on the index page.

File: default_translations/de/messages.de.yml (hunk 5 above)
Comment (doobry-systemli) [reply default_translations/de/messages.de.yml]: You're right, it's not an alert. It's an info box (thus `alert-info`), just like we use it at registration and recovery token. The info box is blue, contrary to the warning (yellow) and error (red) alert boxes.

Comment (0x46616c6b) [reply default_translations/de/messages.de.yml]: I would reuse this alert box from bootstrap and discuss the coloring/contrast/readability later.

**Comment by t2d** [comment]
Should we merge now?

**Pull request merged**